src/ext/coffee-script.ts

```diff
@@ -1,11 +1,12 @@
 import { Base, Op } from 'decaffeinate-coffeescript/lib/coffee-script/nodes';
 
-export function patchCoffeeScript() {
+export function patchCoffeeScript(): void {
   Op.prototype.invert = invert;
   Base.prototype.invert = invert;
 }
 
-function invert() {
+// tslint:disable-next-line no-any
```

Comment:
Yep: `error TS2526: A 'this' type is available only in a non-static member of a class or interface.`